react/components/EXPERIMENTAL_Table/context.ts

```diff
@@ -0,0 +1,7 @@
+import { createContext } from 'react'
+
+const Context = createContext<TableState>(null)
```

Comment:
Interesting! So with this columns, items and isEmpty are accessible anywhere on the table?

Reply:
Yep. This avoids passing props too deep.